### PR TITLE
[JBIDE-22152] - Server Adapter, Debugging: Unknown launch is left in Debug view after deletion of an OS 3 server adapter

### PR DESCRIPTION
**Pull request by jeffmaury**
- Add fake process to OS3 server adapter launch

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java [hunk 1]
```diff
@@ -73,8 +74,17 @@ protected static IControllableServerBehavior getServerBehavior(ILaunchConfigurat
 		IControllableServerBehavior behavior = server.getAdapter(IControllableServerBehavior.class);
 		return behavior;
 	}
-
 	
+	private String getLabel(String mode) {
+	    String label = OpenShiftCoreMessages.RunOnOpenshift;
+	    if (ILaunchManager.PROFILE_MODE.equals(mode)) {
```
Comment (bdshadow) [inline plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java]: just a niggle: i think `switch-case` with "RunOnOpenshift" as the default would be more readable here

**Comment by jeffmaury** [comment]
testPR

**Comment by mickaelistria** [comment]
testPR

**Comment by jeffmaury** [comment]
@adietish @fbricon can you review/merge

**Comment by adietish** [comment]
simple enough, looks good code-wise. Testing it.

**Comment by adietish** [comment]
works for me. Once the adapter is killed the entry turns <terminated> and can be removed.
merging.

**Pull request merged**